src/components/DivElement.js

```diff
@@ -1,6 +1,12 @@
 const HTMLElement = require('./HTMLElement');
 
-// Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super();
+    this.tag = 'div';
+    this.content = content;
```

Comment:
Another way you can do this without having to set tag/content is to call `super()` and pass the tag and content to `super()`. Basically:

```
super('div', content)
```

will call the `HTMLElement` constructor.